public/collections.json

```diff
@@ -64,5 +64,24 @@
       "en": "Online dating",
       "fr": "Rencontre en ligne"
     }
+  },
+  "PGA": {
+    "languages": ["en"],
+    "jurisdictions": ["EU"],
+    "maintainers": [
+      {
+        "name": "Platform Governance Archive",
+        "logo": "/images/contributors/university-of-bremen.png",
+        "url": "https://pga.hiig.de/"
```

Comment:
@ckatzenbach As a _maintainer_ entity, should this be a link to PGA, HIIG, ZeMKI or UBremen?

Reply:
ZeMKI is maintainer and primary host of PGA. Wherever there is space for a second entity, please add HIIG: